superbench/config/amd_mi100.yaml

```diff
@@ -0,0 +1,87 @@
+# SuperBench Config
```

Comment:
AMD_MI100_Z53.ymal and add server link to the configuration file.

Reply:
Done